src/App/App.tsx

```diff
@@ -2,6 +2,7 @@ import { Container } from '@material-ui/core'
 import React from 'react'
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import { ConfigPage } from '../ConfigPage/ConfigPage'
+import { DisplayTotalMasteryPage } from '../MasteryPage/MasteryPage'
```

Comment:
The component itself should also be renamed

```suggestion
import { MasteryPage } from '../MasteryPage/MasteryPage'
```